src/axom/quest/tests/quest_point_in_cell_mfem.cpp

```diff
@@ -390,29 +387,47 @@ class PointInCellTest : public ::testing::Test
 
     // Test that a fixed set of isoparametric coords on each cell
     // maps to the correct place.
-    axom::Array<SpacePt> pts = generateIsoParTestPoints(::TEST_GRID_RES);
+    axom::Array<SpacePt> isoPts = generateIsoParTestPoints(::TEST_GRID_RES);
+
+    const auto SZ = isoPts.size();
+    axom::Array<SpacePt> spacePts(SZ, SZ);
+    axom::Array<SpacePt> foundIso(SZ, SZ);
+    axom::Array<IndexType> foundIDs(SZ, SZ);
+
     axom::utilities::Timer queryTimer2(true);
     SpacePt foundIsoPar;
     for(int eltId = 0; eltId < m_mesh->GetNE(); ++eltId)
     {
-      for(const SpacePt& isoparCenter : pts)
+      // Reconstruct points in space from isoparametric coords on an element
+      for(int idx = 0; idx < SZ; ++idx)
+      {
+        spatialIndex.reconstructPoint(eltId,
+                                      isoPts[idx].data(),
+                                      spacePts[idx].data());
+      }
+
+      // locate the reconstructed points (using EXEC space)
+      spatialIndex.locatePoints(spacePts.view(), foundIDs.data(), foundIso.data());
```

Comment:
Investigating why our CI tests have been really slow for this test...

This call to `locatePoints()` was being called once for each integration point of each element, and has several kernels.
Since `reconstructPoint` is using serial mfem functionality, I converted it to one `locatePoints` per element.

Reply:
By really slow, I mean that the test can take >10 minutes in the CI (sometimes considerably more). 
The test only takes a few seconds when run locally.

Reply:
Seems to have done the trick.
The test is down to a few minutes compared to 10-20 minutes.

E.g. for the clang@10 build:

#### Before
> 273/273 Test #253: quest_point_in_cell_mfem .................   Passed  899.56 sec
https://lc.llnl.gov/gitlab/axom/axom/-/jobs/514998#L6314

#### After
> 259/259 Test #253: quest_point_in_cell_mfem ................   Passed   74.22 sec
https://lc.llnl.gov/gitlab/axom/axom/-/jobs/515255#L7445


As a future optimization, we should consider performing all the point locations in a single call to `locatePoints()`, which would likely bring the results back down to a few seconds.

Reply:
you are so picky @kennyweiss ... 10 minutes is not good enough for you!?